ceph_iscsi_config/lun.py

```diff
@@ -445,6 +460,11 @@ def allocate(self):
                                  "owner": self.owner,
                                  "pool": self.pool,
                                  "pool_id": rbd_image.pool_id}
+                    # add non-default control params to LUN config
+                    for k,v in self.control_dct.iteritems():
+                        val = getattr(settings.config, k, None)
+                        if val is not None and val != v:
```

Comment:
Nit: ```if val and val != v:```

Reply:
The default value can be 0, so the `NoneType` check.